### ISSUE TITLE
Windows docs

### ISSUE THREAD

**Issue by PendragonDevelopment** [Jaicob/rails-docker#1]
We need some work on how to get started on Windows. I'll dig into it, but I definitely need to grok a little better of how Docker works as a whole.

**Comment by Jaicob** [comment]
Agreed, windows is pretty foreign to me but I can try to answer any docker questions you have as they arise.
